participants/christian_hoerauf.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Christian H�rauf",
+  "company": "Consorsbank - BNP Pariabas S.A. Deutschland",
+  "tags": [
+  	"Angular2",
+	"Typescript",
+	"Webpack",
+	"Elm",
+	"Functional Programming",
+	"lodash",
+	"rx.js",
+	"mockinjay.js"
+  ],
+  "when": {
+    "friday": true,
+    "friday_party": true,
+    "saturday": true
+  },
+  "email": "christian.hoerauf@consorsbank.de",
+  "what_is_my_connection_to_javascript": "After being frustrated with C++, I got started into JavaScript and Functional Programming. I love TDD and looking forward for the next technology to turn my head around for 180 degrees.",
+  "what_can_i_contribute": "I have a lot experiance with Angular1 and am currently introducing Angular2 to my company. I also know jasmine very well and developed mocking.js to simplify the generation and usage of mockdata.",
+  "tshirt": "M-XL/",
+  "urls": {
+    "photo": "https://drive.google.com/open?id=0B3-B1iwb9igGWFFsa1RFTGstUzQ",
+    "homepage": "",
```

Comment:
Please simply remove the field instead of leaving it empty, then the tests will passt.